src/main/kotlin/com/mineinabyss/launchy/logic/FabricInstaller.kt

```diff
@@ -72,6 +77,12 @@ object FabricInstaller {
         Utils.writeToFile(profileJsonPath, profileJson.toString())
     }
 
+    suspend fun installBaseConfigs() {
```

Comment:
How come this is in fabric installer, is it related to installing fabric itself at all?